rust-runtime/aws-smithy-runtime-api/src/client/auth/option_resolver.rs

```diff
@@ -3,22 +3,21 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-use crate::client::orchestrator::{
-    AuthOptionResolver, AuthOptionResolverParams, BoxError, HttpAuthOption,
-};
+use crate::client::auth::{AuthOptionResolver, AuthOptionResolverParams, AuthSchemeId};
+use crate::client::orchestrator::BoxError;
 use std::borrow::Cow;
 
 /// New-type around a `Vec<HttpAuthOption>` that implements `AuthOptionResolver`.
 ///
 /// This is useful for clients that don't require `AuthOptionResolverParams` to resolve auth options.
 #[derive(Debug)]
 pub struct AuthOptionListResolver {
-    auth_options: Vec<HttpAuthOption>,
+    auth_options: Vec<AuthSchemeId>,
```

Comment:
- might consider naming this something like `StaticAuthOptionResolver` or something
- should this field be called `scheme_ids` now?

Reply:
> might consider naming this something like StaticAuthOptionResolver or something
That's a good idea. Will do.

> should this field be called scheme_ids now?

They are still the auth options, even if they're represented by scheme IDs.